### ISSUE TITLE
"Channel not found" when trying to get global emotes

### ISSUE THREAD

**Issue by Atlas7005** [mkody/twitch-emoticons#8]
I created a chat application and everything worked fine when it was set up, but suddenly when I fetch global emotes (Twitch & BTTV) I get a channel not found 404 error message.

**Comment by mkody** [comment]
I can confirm that for Twitch, BTTV is still working fine.  

Set is still found https://api.twitchemotes.com/api/v4/sets?id=0
But nothing about the channel https://api.twitchemotes.com/api/v4/channels/0

I'll see if it can be resolved or if I'm just going to host my own service...

**Comment by Atlas7005** [comment]
Yeah, it seems that Twitch Emotes's API has broke, it's probably best if you do your own service, provided it's possible of course.

**Comment by mkody** [comment]
API has been working again for a little while, closing for now.  
I didn't have the time to do a proper replacement, but I have a static copy of the global emotes just in case.

**Comment by mkody** [comment]
Happening again, I'm pushing a temporary fix.
